bin/textlint.js

```diff
@@ -13,6 +14,12 @@ if (useStdIn) {
             process.exit(exitCode);
         });
     }));
+} else if (init) {
+    var configInit = require("../lib/config/config-initializer");
+    configInit.initializeConfig().then(function (code) {
```

Comment:
It should be written in [src/cli.js](https://github.com/textlint/textlint/blob/master/src/cli.js "src/cli.js").

```js
    execute(args, text) {
        var currentOptions;
        try {
            currentOptions = options.parse(args);
        } catch (error) {
            console.error(error.message);
            return Promise.resolve(1);
        }
        const files = currentOptions._;
        if (currentOptions.version) {
            // version from package.json
            console.log(`v${ require('../package.json').version }`);
        } else if (currentOptions.init) {
            // TODO: --init handling 
            return configInit.initializeConfig(process.cwd());
        } else if (currentOptions.help || !files.length && !text) {
            console.log(options.generateHelp());
        } else {
            // specify file name of stdin content
            const stdinFilename = currentOptions.stdinFilename;
            debug(`Running on ${ text ? 'text' : 'files' }, stdin-filename: ${stdinFilename}`);
            return this.executeWithOptions(currentOptions, files, text, stdinFilename);
        }
        return Promise.resolve(0);
    },
```